### PR TITLE
Add google docs

### PR DESCRIPTION
**Pull request by lindsaycarr**
Converted Google doc files to markdown 

See intro-curriculum issues [21](https://github.com/USGS-R/intro-curriculum/issues/21) and [22](https://github.com/USGS-R/intro-curriculum/issues/22)

### CODE REVIEW

File: inst/doc/Crib_sheet.R [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (jread-usgs) [inline inst/doc/Crib_sheet.R]: placeholder too?

Comment (lindsaycarr) [inline inst/doc/Crib_sheet.R]: Nope, there's just not any R code in the markdown file, and `devtools::build_vignettes` creates R script files

File: vignettes/USGS-R_packages.Rmd [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+[EGRETci](https://github.com/USGS-R/EGRETci)
+
+* A bootstrap method for estimating uncertainty of water quality trends
+* http://www.sciencedirect.com/science/article/pii/S1364815215300220
+
+[geoknife](https://github.com/USGS-R/geoknife)
+
+* For web-based geoprocessing of gridded data, especially that available through the USGS Geo Data Portal
+* http://www.ecography.org/article/geoknife-reproducible-web-processing-large-gridded-datasets
```
Comment (jread-usgs) [inline vignettes/USGS-R_packages.Rmd]: dead link. link right to the publication? http://onlinelibrary.wiley.com/doi/10.1111/ecog.01880/abstract

Comment (lindsaycarr) [inline vignettes/USGS-R_packages.Rmd]: didn't realize that was dead - thanks!

File: vignettes/USGS-R_packages.Rmd [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+* http://onlinelibrary.wiley.com/doi/10.1002/rra.2710/abstract 
+
+[rloadest](https://github.com/USGS-R/rloadest)
+
+* For load estimation of constituents in rivers and streams
+* https://github.com/USGS-R/rloadest/tree/master/vignettes 
+
+[gsplot](https://github.com/USGS-R/gsplot)
+
+* SPN-compatible, customizable plotting system for water data
```
Comment (jread-usgs) [inline vignettes/USGS-R_packages.Rmd]: How about just `customizable plotting system for water data` ?

File: vignettes/USGS-R_packages.Rmd [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+* https://github.com/USGS-R/gsplot 
+
+[sbtools](https://github.com/USGS-R/sbtools)
+
+* Tools for interfacing R with ScienceBase data services
+
+[streamMetabolizer](https://github.com/USGS-R/streamMetabolizer)
+
+* Models for estimating stream/river production and respiration from dissolved oxygen data
+* see also LakeMetabolizer (CRAN/GLEON) for lakes
```
Comment (jread-usgs) [inline vignettes/USGS-R_packages.Rmd]: Do you want "CRAN" and "GLEON" to be links here?

**Comment by jread-usgs** [comment]
just had a few comments in there @lindsaycarr

File: vignettes/USGS-R_packages.Rmd (hunk 3 above)
Comment (lindsaycarr) [inline vignettes/USGS-R_packages.Rmd]: wouldn't we like to highlight that you can have it easily meet SPN requirements?

File: vignettes/USGS-R_packages.Rmd (hunk 4 above)
Comment (lindsaycarr) [inline vignettes/USGS-R_packages.Rmd]: I'd rather change that to the github link (which has a statement about CRAN and GLEON)

**Pull request merged**